src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingCohorts.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.ProviderCommitments.Web.Controllers;
+using SFA.DAS.ProviderCommitments.Web.Models.Cohort;
+using SFA.DAS.ProviderUrlHelper;
+
+namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Controllers.CohortControllerTests
+{
+    [TestFixture]
+    public class WhenGettingCohorts 
+    {
+        [Test]
+        public async Task ThenCallsModelMapper()
+        {
+            var fixture = new WhenGettingCohortsFixture();
+
+            await fixture.Act();
+
+            fixture.VerifyMapperWasCalled();
+        }
+
+        [Test]
+        public async Task ThenReturnsView()
+        {
+            var fixture = new WhenGettingCohortsFixture();
+
+            var result = await fixture.Act() as ViewResult;
+
+            Assert.NotNull(result);
+            Assert.AreEqual(typeof(CohortsViewModel), result.Model.GetType());
```

Comment:
Probably the extension method is more readable, but not fussed about it. Leave it to you if you want to change.
`result.VerifyReturnsViewModel().WithModel<CohortsViewModel>();`

Reply:
I can see what you mean, but considering my refactoring of this in CV-714. I'll wait until you make a comment on that one thx

Reply:
moved the comment to CV714. Thx.